### ISSUE TITLE
Improve UI, Re-Position the Send Button.

### ISSUE THREAD

**Issue by 0x48piraj** [fossasia/susi_chromebot#201]
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current UI Design:**

> The **Send** button is very far away, and practically useless. As a **designer** point of view, this is bad!

![image](https://user-images.githubusercontent.com/5800726/33517861-92ed9a0e-d7b1-11e7-8e08-d0ed41963c3c.png)

**Expected behavior:**

It should be more close to the `TextField`

**Steps to reproduce:**
Install and run the `susi_chromebot`

**Other information:**
https://github.com/fossasia/susi_chromebot#installation

**System information:**

```
Your operating system : Windows
```

**Would you like to work on the issue?**

**Yep.**

**Comment by ms10398** [comment]
It shows correctly in my Computer and almost for everyone check once again!

**Comment by zamhaq** [comment]
@0x48piraj how's that possible? We did correct it. Maybe positions are fixed value and that is causing this issue. Please mention your browser version. 😅

**Comment by 0x48piraj** [comment]
![image](https://user-images.githubusercontent.com/5800726/33518135-f867e6c4-d7b5-11e7-8b29-f3dcec7760da.png)

Maybe I should add check on `bug` checkbox too :wink:

**Comment by zamhaq** [comment]
@0x48piraj yeah. You can work on it ☺️ see what's causing the issue and what can be done.

**Comment by 0x48piraj** [comment]
@zamhaq Please approve my task on GCI!

> I will start debugging in night :wind_chime:

**Comment by stealthanthrax** [comment]
@0x48piraj , what's the status ?

**Comment by 0x48piraj** [comment]
@stealthanthrax 
I debugged the **chromebot**, but it didn't helped me much, as the bug out of nowhere pops in this specific version of chrome.
I think, I should start debugging the chrome browser itself to dig deeper.
I'm new to software debugging, setting breakpoints/faults, so It's taking time. :worried:

**Comment by stealthanthrax** [comment]
@0x48piraj , try using relative widths or adding media queries for a device resolution greater than a certain value.

**Comment by zamhaq** [comment]
@0x48piraj have you tried fixing this? Are you still facing the UI issues as before? Can you update your fork and check? Thanks.

**Comment by 0x48piraj** [comment]
I'll update my fork and will test again!
